RestCommClient/Classes/RCDevice.h

```diff
@@ -126,7 +126,10 @@ extern NSString* const RCDeviceCapabilityClientNameKey;
  *    <b>registrar</b>: Restcomm instance to use, like <i>'cloud.restcomm.com'</i>. Leave empty for registrar-less mode <br>
  *    <b>signaling-secure</b>: Should signaling traffic be encrypted? For traffic to be encrypted the 'signaling-certificate-dir' below should be provided as well <br>
  *    <b>signaling-certificate-dir</b>: Directory where files agent.pem and cafile.pem are installed inside the App bundle needed when TLS is to be used for signaling. agent.pem contains the private key and certificate to be used by the signaling facilities. cafile.pem is a file containing all trusted certs
-*     <b>stun-turn-servers</b> NSArray of NSDictionary data of the STUN and TURN servers; This key is mandatory if ICEConfigType is 'kCustom'; Example with 2 servers:
+ *    <b>ice-config-type<b>: Should be integer value of the following  kXirsysV2 = 0, kXirsysV3 =1 , kCustom=2
+      for example, if we want to use XirsysV2 server value should be 1
+
+ *    <b>ice-servers</b> NSArray of NSDictionary data of the STUN and TURN servers; This key is mandatory if ice-config-type is 'kCustom'; Example with 2 servers:
```

Comment:
@ognjenns probably typo? There's just 1 server here and it's missing the "credential" key. I would suggest that you use an example where you have 2 servers, where one of them is STUN (that typically doesn't contain username/password, as it's kind of free) and then other is a TURN server containing everything (and use literal to make easier to see). For example you can use this:

```
@[
   @{ @"url" : @"stun:turn01.uswest.xirsys.com" },
   @{ @"url" : @"turn:turn01.uswest.xirsys.com:80?transport=udp", @"username" : @"USERNAME_HERE", @"credential" : @"KEY_HERE" }
]
```

Reply:
What i did here is that i took the response and see in which format we are getting the list of servers (that curl you sent me). I have put the example in the comment.

Reply:
done